src/hackerspace_online/tests/test_management_commands.py

```diff
@@ -85,3 +89,25 @@ def test_initdb(self):
             self.assertTrue(Site.objects.exists())
 
             Tenant.objects.get(schema_name=apps.get_app_config('library').TENANT_NAME)  # no assert, but will throw exception if doesn't exist
+
+
+class GenerateContentTest(TenantTestCase, CommandMixin):
+    """ generate adds items to an existing tenant.
+    Dont need extensive testing as tests exist in "test_shell_utils.py"
+    """
+    name = "generate_content"
+
+    def test_added_rows_to_db(self):
+        """ test checks if "generate_content" adds objects to the db """
+        # cache initial numbers
+        quest_count = Quest.objects.count()
+        campaign_count = Category.objects.count()
+        user_count = User.objects.count()
+
+        # name is in a list?
+        self.call_command(self.tenant.schema_name, '--quiet')
```

Comment:
Maybe I'm confused, but I don't see where generate_content command is called here?  Does the mixin use the name variable?  I think it would be more clear to pass the name as a parameter.

Reply:
the command name is called by ```call_command``` using ```self.name```. Its similar to the format used by Django's ```TemplateViewMixin``` where you specify the ```template_name``` in the child class and its used by the mixin.
![image](https://github.com/bytedeck/bytedeck/assets/39788517/387c0c3c-0b43-4ce4-86a7-98e2c32b6236)
![image](https://github.com/bytedeck/bytedeck/assets/39788517/d5400079-24c2-4232-a5c7-32fae8322b4c)

Reply:
oh and heres where ```self.call_command``` is knowing what the name of the command is
![image](https://github.com/bytedeck/bytedeck/assets/39788517/878e3070-a7cc-4317-907a-cf6ef2c9a960)

Reply:
Ok, sounds good. Please improve the docstring of the Mixin to include `name` variable explicitly

Reply:
done

Reply:
added as

```
need to set name variable in the class that uses this mixin.
    ie.
    ```
        def CustomCommandTest(CommandMixin):
            name = "custom_command"
    ```
```